PreFreeSurfer/PreFreeSurferPipeline.sh

```diff
@@ -452,253 +517,311 @@ log_Msg "POSIXLY_CORRECT="${POSIXLY_CORRECT}
-		--outbrainmask=${TXwFolder}/${TXwImage}_acpc_brain_mask \
-		--fnirtconfig=${FNIRTConfig}
-
-done
-
-# End of looping over modalities (T1w and T2w)
-
-# ------------------------------------------------------------------------------
-#  T2w to T1w Registration and Optional Readout Distortion Correction
-# ------------------------------------------------------------------------------
-
-case $AvgrdcSTRING in
-
-	${FIELDMAP_METHOD_OPT} | ${SPIN_ECHO_METHOD_OPT} | ${GENERAL_ELECTRIC_METHOD_OPT} | ${SIEMENS_METHOD_OPT})
-
-		log_Msg "Performing ${AvgrdcSTRING} Readout Distortion Correction"
-		wdir=${T2wFolder}/T2wToT1wDistortionCorrectAndReg
-		if [ -d ${wdir} ] ; then
-			# DO NOT change the following line to "rm -r ${wdir}" because the
-			# chances of something going wrong with that are much higher, and
-			# rm -r always needs to be treated with the utmost caution
-			rm -r ${T2wFolder}/T2wToT1wDistortionCorrectAndReg
-		fi
-
-		log_Msg "mkdir -p ${wdir}"
-		mkdir -p ${wdir}
-
-		${RUN} ${HCPPIPEDIR_PreFS}/T2wToT1wDistortionCorrectAndReg.sh \
-			--workingdir=${wdir} \
-			--t1=${T1wFolder}/${T1wImage}_acpc \
-			--t1brain=${T1wFolder}/${T1wImage}_acpc_brain \
-			--t2=${T2wFolder}/${T2wImage}_acpc \
-			--t2brain=${T2wFolder}/${T2wImage}_acpc_brain \
-			--fmapmag=${MagnitudeInputName} \
-			--fmapphase=${PhaseInputName} \
-			--fmapgeneralelectric=${GEB0InputName} \
-			--echodiff=${TE} \
-			--SEPhaseNeg=${SpinEchoPhaseEncodeNegative} \
-			--SEPhasePos=${SpinEchoPhaseEncodePositive} \
-			--seechospacing=${SEEchoSpacing} \
-			--seunwarpdir=${SEUnwarpDir} \
-			--t1sampspacing=${T1wSampleSpacing} \
-			--t2sampspacing=${T2wSampleSpacing} \
-			--unwarpdir=${UnwarpDir} \
-			--ot1=${T1wFolder}/${T1wImage}_acpc_dc \
-			--ot1brain=${T1wFolder}/${T1wImage}_acpc_dc_brain \
-			--ot1warp=${T1wFolder}/xfms/${T1wImage}_dc \
-			--ot2=${T1wFolder}/${T2wImage}_acpc_dc \
-			--ot2warp=${T1wFolder}/xfms/${T2wImage}_reg_dc \
-			--method=${AvgrdcSTRING} \
-			--topupconfig=${TopupConfig} \
-			--gdcoeffs=${GradientDistortionCoeffs} \
-			--usejacobian=${UseJacobian}
-
-		;;
-
-	*)
-
-		log_Msg "NOT PERFORMING READOUT DISTORTION CORRECTION"
-		wdir=${T2wFolder}/T2wToT1wReg
-		if [ -e ${wdir} ] ; then
-			# DO NOT change the following line to "rm -r ${wdir}" because the
-			# chances of something going wrong with that are much higher, and
-			# rm -r always needs to be treated with the utmost caution
-			rm -r ${T2wFolder}/T2wToT1wReg
-		fi
-
-		log_Msg "mkdir -p ${wdir}"
-		mkdir -p ${wdir}
-
-		${RUN} ${HCPPIPEDIR_PreFS}/T2wToT1wReg.sh \
-			${wdir} \
-			${T1wFolder}/${T1wImage}_acpc \
-			${T1wFolder}/${T1wImage}_acpc_brain \
-			${T2wFolder}/${T2wImage}_acpc \
-			${T2wFolder}/${T2wImage}_acpc_brain \
-			${T1wFolder}/${T1wImage}_acpc_dc \
-			${T1wFolder}/${T1wImage}_acpc_dc_brain \
-			${T1wFolder}/xfms/${T1wImage}_dc \
-			${T1wFolder}/${T2wImage}_acpc_dc \
-			${T1wFolder}/xfms/${T2wImage}_reg_dc
-
-esac
+# --- skip if we have a custom brain
```

Comment:
Slightly more specific comment would be helpful here to help readers appreciate the large scope of the following conditional
```
# NOTE: We skip all the way to AtlasRegistration (last step) if using a custom brain mask
# or custom structural images ($CustomBrain={MASK|CUSTOM})
```

Reply:
Done.